support-metrics-client/src/test/java/io/confluent/support/metrics/MetricsToKafkaTest.java

```diff
@@ -122,33 +122,23 @@ public void retrievesBasicMetricsSubmittedByMultiNodeCluster() throws IOExceptio
    * Helper function that consumes messages from a topic with a timeout.
    */
   private static void verifyMetricsSubmittedToTopic(
-      String zkConnect,
+      String bootstrapServer,
       String topic,
       int expNumMetricSubmissions) throws IOException {
     int timeoutMs = 10 * 1000;
-    KafkaMetricsToFile kafkaMetricsToFile = new KafkaMetricsToFile(zkConnect, timeoutMs);
-    List<KafkaStream<byte[], byte[]>> streams = kafkaMetricsToFile.getStreams(topic);
+    KafkaMetricsToFile kafkaMetricsToFile = new KafkaMetricsToFile(bootstrapServer);
+    KafkaConsumer<byte[], byte[]> consumer = kafkaMetricsToFile.createConsumer();
+    consumer.subscribe(singleton(topic));
 
-    int numRecords = 0;
+    Collection<ConsumerRecord<byte[], byte[]>> records = JavaConverters.asJavaCollectionConverter(
+            TestUtils.consumeRecords(consumer, expNumMetricSubmissions, timeoutMs)).asJavaCollection();
```

Comment:
Do we use `TestUtils.consumeRecords`in other tests? I was thinking it would just be a few lines of code to avoid that?

Reply:
Because this code is so coupled to the AK code anyway, I thought it was OK for now.